src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -297,12 +297,15 @@ public FormValidation doCheckRepositorySlug(@QueryParameter String value) {
         @POST
         public FormValidation doCheckServerId(@QueryParameter String serverId) {
             Jenkins.get().checkPermission(Permission.CONFIGURE);
-            List<BitbucketServerConfiguration> serverList =
-                    new BitbucketPluginConfiguration().getServerList();
+            BitbucketPluginConfiguration pluginConfiguration = new BitbucketPluginConfiguration();
```

Comment:
As discussed in IRL, let's rebase on `master` and use the injected `BitbucketPluginConfiguration` instead of creating a new instance